### DISCUSSION TITLE
netbox prefixes view

### DISCUSSION THREAD

**Discussion by lasag-2511** [netbox-community/netbox#13896] [General]
Hi all,

When I add the ipaddresses initally, it was showing both active IPs and if the ips are available, the available ips were listed  between active ips. Now the GUI shows all aavailable ips at the end. Is this a normal behaviour ?

![available](https://github.com/netbox-community/netbox/assets/38073302/94eab556-5e16-416c-a768-5605fe164458)

**Comment by mathewbishop** [comment]
Hey @lasag-2511,

Are you sure there are active IPs tied to this prefix you're viewing?

Here is a `/30` prefix from my Netbox instance, which shows one active IP and one available:

<br/>


![image](https://github.com/netbox-community/netbox/assets/39043451/447658fe-8df9-47f8-b503-03581c3d1006)


<br/>
<br/>

Does this help? Am I misunderstanding?

**Comment by royvegard** [comment]
Hi, @lasag-2511 and @mathewbishop 

What versions of netbox are these screenshots from? I'm on version 3.6.4 and have also noticed that the Prefixes > IP addresses view no longer shows the available addresses. It only shows the used addresses in the prefix and does not show the green lines with available IPs. I suspect that this might be a recent change and maybe @mathewbishop's screenshot is from before that change.

**Reply by royvegard** [reply]
It looks like the IP available lines disappear when an ordering is applied to the table. And to get them back you just have to clear the ordering (by clicking the little red X that appears on the ordered column name). Ref #13913

**Reply by mathewbishop** [reply]
@royvegard to answer your question, I'm running v3.5.7
